cmd/minersc.go

```diff
@@ -264,12 +291,30 @@ var minerscSharders = &cobra.Command{
 		if allFlag {
 			sharders := new(zcncore.MinerSCNodes)
 			callback := NewJSONInfoCB(sharders)
-			if err = zcncore.GetSharders(callback); err != nil {
-				log.Fatalf("Failed to get registered sharders: %v", err)
+
```

Comment:
We need to iterate to get all when `--all` is set

Reply:
@peterlimg Then limit and offset fields should not matter.
So should we throw error in such case?

Reply:
@K-Kumar-01 the limit and offset are for pagination so that we would not have big requests/response. But we will need to be able to get all if `--all` option is set. Or we need to remove the `--all` option, but that may break a lot of test cases.

Reply:
https://github.com/0chain/zboxcli/pull/359